plugins/tracker-resources/src/components/issues/Issues.svelte

```diff
@@ -50,19 +53,25 @@
   export let shouldShowEmptyGroups: boolean | undefined = false
   export let includedGroups: Partial<Record<IssuesGroupByKeys, Array<any>>> = {}
 
+  const dispatch = createEventDispatcher()
   const ENTRIES_LIMIT = 200
   const spaceQuery = createQuery()
   const issuesQuery = createQuery()
   const resultIssuesQuery = createQuery()
   const statusesQuery = createQuery()
   const issuesMap: { [status: string]: number } = {}
+
+  let filterElement: HTMLElement | null = null
+  let filters: { [p: string]: any[] } = {}
```

Comment:
Why not have array of filters?
I would prefer to have arrays with typed DocumentQuery inside.
Also assume you will need to add support for $or as next step.